src/utils.js

```diff
@@ -8,3 +8,31 @@ exports.findHighest = (...numbers) => {
     return highest;    
 };
 
+
+exports.findAverage = (...numbers) => {
+  let ave = 0;
+  let res = numbers.every((element) => {return Number.isInteger(element);});
+  
+  try {
```

Comment:
There is no need for the try/catch. Let an exception be thrown

Reply:
noted